src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx

```diff
@@ -28,38 +28,23 @@ function AimIntegrations() {
 
   const integrations = [
     {
-      title: 'Integrate PyTorch Lightning',
```

Comment:
Please don't remove PyTorch Lightning! This is the most popular ML framework!
You could instead add a (regular) PyTorch section.

Reply:
done -- the mlflow autologging docs refer to "Pytorch" but the example they give is Pytorch Lightening, so not adding a regular PyTorch section.